scripts/generate-device-ca-certs.sh

```diff
@@ -2,12 +2,15 @@
 
 source "$(cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )/cert-utils.sh"
 
-ENV_FILE="/etc/sysconfig/go-fdo-server-device-ca"
-[ ! -f "${ENV_FILE}" ] || source "${ENV_FILE}"
-
-conf_dir="${DEVICE_CA_CONF_DIR:-/etc/go-fdo-server}"
-subj="${DEVICE_CA_SUBJECT:-/C=US/O=FDO/CN=Device CA}"
-key="${DEVICE_CA_KEY:-${conf_dir}/device-ca.key}"
-crt="${DEVICE_CA_CRT:-${conf_dir}/device-ca.crt}"
+cert_dir="/etc/pki/go-fdo-server"
+subj="/C=US/O=FDO/CN=Device CA"
+key="${cert_dir}/device-ca-example.key"
+crt="${cert_dir}/device-ca-example.crt"
 
+# Do not overwrite existing cert/key files unless one of the pair is
+# missing
+if [[ ! -f "${key}" || ! -f "${crt}" ]]; then
+  rm -f "${key}" "${crt}"
+fi
 generate_cert "${key}" "${crt}" "${subj}"
+chmod g+r "${key}"
```

Comment:
The device CA private key must belong and it must be readable by the manufacturer user only as it's the only one using it for signing. 

The device CA certificate must belong to the manufacturer user and be readable by all the services so the group should be `go-fdo-server` and the group should be able to read the file.